packages/wallet/test/KeyManagement/InMemoryLedgerKeyAgent.test.ts

```diff
@@ -0,0 +1,27 @@
+// TODO - Used just for fast checks
+/* import { KeyManagement } from '../../src';
+import { TransportType } from '../../src/KeyManagement/InMemoryLedgerKeyAgent';
+
+describe('InMemoryLedgerKeyAgent', () => {
```

Comment:
```suggestion
describe('LedgerKeyAgent', () => {
```

Reply:
I will completely remove this file since is not relevant anymore